### ISSUE TITLE
Seperate gradients in tf.gradients

### ISSUE THREAD

**Issue by mbz** [tensorflow/tensorflow#7296]
based on [documentation](https://www.tensorflow.org/api_docs/python/train/gradient_computation#gradients) `tf.gradients` return the `sum(dy/dx)`. For our problem, **sum** doesn't make sense. Is it possible to get the list of gradients instead?

**Comment by yaroslavvb** [comment]
Generally not possible, you need to do `n` calls to `tf.gradients` or use a trick from ian here: https://github.com/tensorflow/tensorflow/issues/4897#issuecomment-253056458

**Comment by yaroslavvb** [comment]
Closing to keep things in one place (http://stackoverflow.com/questions/42058690/separate-gradients-in-tf-gradients)
